core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.ts

```diff
@@ -82,6 +82,9 @@ export class OperatorPropertyEditFrameComponent implements OnInit, OnChanges, On
   formlyFields: FormlyFieldConfig[] | undefined;
   formTitle: string | undefined;
 
+  // The field name keys within this set will be highlighted, e.g. for showing the diff between two workflows.
+  fieldNamesToHighlight: Set<String> = new Set();
```

Comment:
I do not see API where the user of this feature can specify which fields to be highlighted? The set here seems to be hardcoded?